composeml/label_maker.py

```diff
@@ -106,16 +109,17 @@ def df_to_labels(df):
             labels.index = labels.index.astype('datetime64[ns]')
             return labels.to_frame()
 
+        df = self._preprocess(df)
+        labels = df.groupby(self.target_entity)
+        progress_bar = None
+
         if verbose:
             bar_format = "Elapsed: {elapsed} | Remaining: {remaining} | "
             bar_format += "Progress: {l_bar}{bar}| "
             bar_format += self.target_entity + ": {n}/{total} "
-            tqdm.pandas(bar_format=bar_format, ncols=90)
+            progress_bar = tqdm(total=labels.ngroups, bar_format=bar_format, ncols=90)
```

Comment:
rather than use `verbose`, you can pass the `disable=verbose` parameter to tqdm. that way it will automatically not show if verbose is false and you don't need special verbose logic

Reply:
Didn't know that. I made the update.